File: frontend/src/app/ner-annotator/ner-jobs/ner-job.dto.ts
export class NerJobDto {

    private _id: number;
    private _name: string;
    private _created: Date;


    get id(): number {
        return this._id;
    }

    set id(value: number) {
        this._id = value;
    }

    get name(): string {
        return this._name;
    }

    set name(value: string) {
        this._name = value;
    }

    get created(): Date {
        return this._created;
    }

    set created(value: Date) {
        this._created = value;
    }
}

File: src/main/java/com/pozpl/nerannotator/service/ner/jobmanagement/NerJobDto.java
package com.pozpl.nerannotator.service.ner.jobmanagement;

import com.pozpl.nerannotator.service.ner.jobmanagement.labels.NerLabelDto;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Date;
import java.util.List;

@Builder(toBuilder = true)
@Data
@NoArgsConstructor
@AllArgsConstructor
public class NerJobDto {

	private Integer id;

	private String name;

	private Date created;

	private List<NerLabelDto> labels;
	
}
